workflow/automation/execution_scripts/query_mgmt_db.py

```diff
@@ -68,6 +70,8 @@ def state_table_query_builder(
         wheres.append(f"run_name = ?")
     elif run_name_similar:
         wheres.append(f"run_name like ?")
+    elif run_name_disimilar:
```

Comment:
Can these run_name_exact / similar and dissimilar be a single variable that's passed in instead of 3 individual ones? Maybe use the ComparisonOperator Enum here to replace =, like and not like in the f string.

Reply:
Good point